CMakeLists.txt

```diff
@@ -599,7 +599,9 @@ endif()
 if(WITH_BOOST_CONTEXT)
   list(APPEND BOOST_COMPONENTS context coroutine)
 endif()
-list(APPEND BOOST_COMPONENTS test timer)
+if(NOT FREEBSD)
+  list(APPEND BOOST_COMPONENTS test)
```

Comment:
@wjwithagen `test` is added to this list only if WITH_SEASTAR is ON, but we don't need it anymore, see #22521. could you help review/test it?